example/package.json

```diff
@@ -2,9 +2,9 @@
 	"scripts": {
 		"dev": "npm run build:untouched && npm run serve",
 		"build": "npm run build:untouched && npm run build:optimized",
-		"serve": "serve .",
-		"build:untouched": "asc assembly/index.ts --target debug --exportRuntime",
-		"build:optimized": "asc assembly/index.ts --target release --exportRuntime",
+		"serve": "serve ../",
```

Comment:
We should serve `./` so that the example is fully standalone not relying on the parent folder, to prove it works the same way it works if someone else imports asdom into their own project and they don't have the same parent folder.

Reply:
ok